.github/workflows/main.yml

```diff
@@ -55,9 +54,8 @@ jobs:
       matrix:
         # Github Actions requires a single row to be added to the build matrix.
         # See https://help.github.com/en/articles/workflow-syntax-for-github-actions.
-        name: [
-          windows-clang-cl,
-        ]
+        name: 
+          - windows-clang-cl
```

Comment:
For the merge conflict you can just remove the `name:` here, the `include:` already takes care of everything.